rfcs/0002-authentication.md

```diff
@@ -1,12 +1,12 @@
 ---
-RFC: 0002
+RFC: 2
```

Comment:
@zdne Stephen stated how it is working in this PR. Do you mean something different?